CHANGELOG.rst

```diff
@@ -5,6 +5,10 @@ These are notable changes in edx-platform.  This is a rolling list of changes,
 in roughly chronological order, most recent first.  Add your entries at or near
 the top.  Include a label indicating the component affected.
 
+LMS: Assign cohorts to content groups in the instructor dashboard.  TNL-653
```

Comment:
@cahrens @dan-f @jimabramson We should fix up the CHANGELOG to reflect everything that we've done (or just have a single line entry for cohorted courseware).